doc: Remove documentation for child/parent image (#20005)
nrfconnect/sdk-nrf: doc/nrf/app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_config.rst

Using custom project configurations
***********************************

> You can use custom project configuration options for the associated image, specifying them at build time, temporarily until you clean the build pristinely.

Reviewer: ```suggestion
You can use custom project configuration options for the associated image, specifying them at build time. 
These settings are temporary and will remain enabled until you clean the build pristinely.
```

   -Dmcuboot_FILE_SUFFIX=upgradable \
   -Dapp_FILE_SUFFIX=app

You can use custom project configuration files in combination with temporary configuration options associated with a single build, set using either the command line or Kconfig fragments.

Assigning Kconfig fragments
***************************

You can use Kconfig fragments specific to bootloaders to set temporary configuration options for the associated image, specifying them at build time.

For example, you can assign the :file:`my-custom-fragment.conf` fragment to the |NSIB|, which uses the image name of ``b0``, as follows:

.. code-block:: console

   west build -b nrf52840dk/nrf52840 zephyr/samples/hello_world -- \
   -DSB_CONFIG_SECURE_BOOT_APPCORE=y \
   -DSB_CONFIG_BOOTLOADER_MCUBOOT=y \
   -Db0_EXTRA_CONF_FILE=my-custom-fragment.conf

In the same way, you can replace ``b0`` with ``mcuboot`` to apply the :file:`my-custom-fragment.conf` fragment to the MCUboot image:

.. code-block:: console

   west build -b nrf52840dk/nrf52840 zephyr/samples/hello_world -- \
   -DSB_CONFIG_SECURE_BOOT_APPCORE=y \
   -DSB_CONFIG_BOOTLOADER_MCUBOOT=y \
   -Dmcuboot_EXTRA_CONF_FILE=my-custom-fragment.conf

> You can use this method to apply Kconfig fragments to any image in the build, as well as to set any Kconfig option that can be set from the command line.

Reviewer: ```suggestion
You can use this method to apply Kconfig fragments to any image in the build, and set any Kconfig option that is available from the command line.
```
